website/lookup.php

```diff
@@ -0,0 +1,93 @@
+<?php
+	@define('CONST_ConnectionBucket_PageType', 'Reverse');
+
+	require_once(dirname(dirname(__FILE__)).'/lib/init-website.php');
+	require_once(CONST_BasePath.'/lib/log.php');
+	require_once(CONST_BasePath.'/lib/PlaceLookup.php');
+
+	if (strpos(CONST_BulkUserIPs, ','.$_SERVER["REMOTE_ADDR"].',') !== false)
+	{
+		$fLoadAvg = getLoadAverage();
+		if ($fLoadAvg > 2) sleep(60);
+		if ($fLoadAvg > 4) sleep(120);
+		if ($fLoadAvg > 6)
+		{
+			userError("Bulk User: Temporary block due to high server load");
+			exit;
+		}
+	}
+
+	$oDB =& getDB();
+	ini_set('memory_limit', '200M');
+
+	// Format for output
+	$sOutputFormat = 'xml';
+	if (isset($_GET['format']) && ($_GET['format'] == 'xml' || $_GET['format'] == 'json'))
+	{
+		$sOutputFormat = $_GET['format'];
+	}
+
+	// Show address breakdown
+	$bShowAddressDetails = true;
+	if (isset($_GET['addressdetails'])) $bShowAddressDetails = (bool)$_GET['addressdetails'];
+
+	// Preferred language
+	$aLangPrefOrder = getPreferredLanguages();
+
+	$hLog = logStart($oDB, 'place', $_SERVER['QUERY_STRING'], $aLangPrefOrder);
+
+	$aSearchResults = array();
+	if (isset($_GET['osm_ids']))
+	{
+		$oPlaceLookup = new PlaceLookup($oDB);
+		$oPlaceLookup->setLanguagePreference($aLangPrefOrder);
+		$oPlaceLookup->setIncludeAddressDetails($bShowAddressDetails);
+		
+		$osm_ids = explode(',', $_GET['osm_ids']);
```

Comment:
Minor complaint about style: please continue the variable style of camel case and tagging with the type (i.e. $aOsmIds) in php files. Also for $type and $id below.